API/Controllers/ElectricityController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using API.Data;
+using API.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace trackingapi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ElectricityController : ControllerBase
+    {
+        private readonly IssueDbContext _context;       //Allow controller to access the database
```

Comment:
Comment must be /* */!!!11!! Follow coding style pls!!!!!!!!

Reply:
Fr

Reply:
> Fr

no cap

Reply:
Sorry and thanks for the note! Already fixed.

Reply:
ur welcome